_rules/link-non-empty-accessible-name-c487ae.md

```diff
@@ -27,7 +26,7 @@ accessibility_requirements:
     passed: further testing needed
     inapplicable: further testing needed
   wcag20:1.1.1: # Non-text content (A)
-    secondary: true
+    secondary: This success criterion **partially overlaps** this rule. This is because the HTML `area` element counts as both a link and as non-text content. This is why some of the failed examples may not satisfy this success criterion.
```

Comment:
```suggestion
    secondary: This success criterion is **conditionally applicable** in this rule. It is applicable because the HTML `area` element counts as both a link and as non-text content. This is why some of the failed examples may not satisfy this success criterion.
```

Reply:
I didn't see how an explanation would be written for [Keyboard Trap Example 7](https://w3c.github.io/wcag-act/act-rules-format.html#example-7be89ba3) in this PR, but I thought "partially" would work better there than here. 

For Ex.7: This SC is **partially covered** in this rule because the rule does not test for other possible solutions. As a result, this SC may be satisfied in failed examples.